ModelicaTest/MultiBody.mo

```diff
@@ -3734,14 +3734,14 @@ a linear damper is connected here.
         v(fixed=true),
         n={0,1,0},
         s(start=-1, fixed=true),
-        a(fixed=true, start=-0.81)) annotation (Placement(transformation(
+        a(fixed=true, start=9-world.g)) annotation (Placement(transformation(
             extent={{-10,-10},{10,10}},
             rotation=270,
             origin={-4,16})));
     equation
-      assert(body.r_0[2] - body.r_0[2] < tol,
+      assert(body.r_0[2] - body1.r_0[2] < tol,
```

Comment:
```suggestion
      assert(abs(body.r_0[2] - body1.r_0[2]) < tol,
```
We should test that they are close enough.

Reply:
This concerns far more examples. :-|
Will try to modify them all.